wger/nutrition/views/ingredient.py

```diff
@@ -56,19 +57,35 @@ def get_queryset(self):
         (the user can also want to see ingredients in English, in addition to
         his native language, see load_ingredient_languages)
         '''
-        languages = load_ingredient_languages(self.request)
-        return (Ingredient.objects.filter(language__in=languages)
-                .filter(status__in=Ingredient.INGREDIENT_STATUS_OK).only(
-                    'id', 'name'))
+        language = None
+        lg_code = self.request.GET.get('lang', None)
```

Comment:
Please use descriptive variable names, a name such as `lg_code` is quite ambigious.

Reply:
@parseendavid  I have made changes to use a descriptive name `language_code`